### PR TITLE
clang format

### PR DESCRIPTION
**Pull request by kendase3**

### CODE REVIEW

**Comment by kendase3** [comment]
the build failed:
```devmapper: Thin Pool has 163184 free data blocks which is less than minimum required 163840 free data blocks. Create more free space in thin pool or use dm.min_free_space option to change behavior```

File: src/metric.cc [hunk 1]
```diff
@@ -105,7 +105,7 @@ static QString getUuid() {
 // the metrics layer stores the site id to provide in later metrics
 // once it is available
 namespace {
-  static int site_id = 0;
```
[the remark this answers is not in the document]
Comment (kendase3) [reply src/metric.cc]: plz use single line comment instead of review for comments like this so it's clear the PR can be merged

**Comment by kendase3** [comment]
retest this plz

**Comment by nicholasbishop** [comment]
Old Docker stuff still in the process of being deleted from Jenkins. There seems to be a lot of it.

**Comment by nicholasbishop** [comment]
Tests passed this time

**Comment by kendase3** [comment]
i really worked for this PR imo

**Pull request merged**